core/controller/src/whisk/core/controller/Actions.scala

```diff
@@ -387,21 +388,25 @@ trait WhiskActionsApi extends WhiskCollectionAPI {
         // merge package parameters with action (action parameters supersede), then merge in payload
         val args = { env map { _ ++ action.parameters } getOrElse action.parameters } merge payload
         val message = Message(transid, s"/actions/invoke/${action.namespace}/${action.name}/${action.rev}", user, ActivationId(), args)
-        info(this, s"[POST] action activation id: ${message.activationId}")
+        // TODO keep this info msg as is and extract the activation id
+        marker(this, CONTROLLER_CREATE_ACTIVATION, s"[POST] action activation id: ${message.activationId}")
         performLoadBalancerRequest(INVOKER, message, transid) map {
             (action.limits.timeout(), _)
         } flatMap {
             case (duration, response) =>
                 response.id match {
                     case Some(activationId) =>
+                        marker(this, CONTROLLER_ACTIVATION_END)
                         Future successful (duration, activationId)
                     case None =>
                         if (response.error.getOrElse("??").equals("too many concurrent activations")) {
                             // got here because of DoS throttle
                             warn(this, s"[POST] action activation rejected: ${response.error.getOrElse("??")}")
+                            marker(this, CONTROLLER_ACTIVATION_REJECTED)
                             Future failed new TooManyActivationException("too many concurrent activations")
                         } else {
                             error(this, s"[POST] action activation failed: ${response.error.getOrElse("??")}")
+                            marker(this, CONTROLLER_ACTIVATION_FAILED)
```

Comment:
This will generate two log messages - the error message and the marker. Should marker interface accept an optional level (default INFO)? Error messages are always printed regardless of verbosity level. This would avoid two log messages. I would apply use the generalized model for `warn` as well (in which case the marker is emitted but not the warning message unless the verbosity threshold is crossed).